jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHttpOutputInterceptor.java

```diff
@@ -40,6 +39,8 @@
 import org.eclipse.jetty.util.log.Log;
 import org.eclipse.jetty.util.log.Logger;
 
+import static org.eclipse.jetty.http.CompressedContentFormat.GZIP;
+
 public class GzipHttpOutputInterceptor implements HttpOutput.Interceptor
```

Comment:
Hmmm naming issues with all the gzip in the package and class names.... not sure what to do.... ignore for now